ethsigner/signer/multikey/src/test/java/tech/pegasys/ethsigner/signer/multikey/MultiKeyTransactionSignerProviderTest.java

```diff
@@ -46,7 +46,7 @@
   private SigningMetadataTomlConfigLoader loader = mock(SigningMetadataTomlConfigLoader.class);
   final AzureKeyVaultTransactionSignerFactory azureFactory =
       new AzureKeyVaultTransactionSignerFactory(new AzureKeyVaultAuthenticator());
-  private final HashicorpSignerFactory hashicorpFactory = new HashicorpSignerFactory();
+  private final HashicorpSigner hashicorpFactory = new HashicorpSigner();
```

Comment:
nit: variable vs class naming

Reply:
Why did @macfarla changed to HashicorpSigner? It was appropriately named as HashicorpSignerFactory .... we have FileBasedSignerFactory, AzureKeyVaultTransactionSignerFactory and should have HashicorpVaultSignerFactory

Reply:
Resolving conversation as it has been renamed back to HashicorpVaultSignerFactory.